packages/react-components/react-avatar/src/utils/partitionAvatarGroupItems.ts

```diff
@@ -28,5 +37,6 @@ export const partitionAvatarGroupItems = <T>(options: PartitionAvatarGroupItemsO
   return {
     inlineItems: items.slice(inlineCount),
     overflowItems: items.slice(0, inlineCount),
+    renderOverflowButton: items.length > maxInlineItems,
```

Comment:
Could you just have `overflowItems` be undefined in this case, instead of a separate `renderOverflowButton` variable?

Reply:
sadly no, when this is the case: `<AvatarGroupPopover>{undefined}</AvatarGroupPopover>`, AvatarGroupPopover is still rendered with +0.

Reply:
My previous approach made it so AvatarGroupPopover took care of the case where a `[] | undefined | null` wouldn't render, but I think this should be handled by the user instead.

Reply:
What made you decide users should handle the conditional rendering rather than `AvatarGroupPopover`? It seemed nice to me that the popover took care of this behavior.

Reply:
I don't think it makes sense for AvatarGroupPopover to handle it, its behavior is correct. If it receives nothing, there's +0 overflowing items. I'm open to go back to that behavior, but it just seems like AvatarGroupPopover sometimes not rendering is wrong.